src/local-storage.js

```diff
@@ -545,7 +545,7 @@ LocalStorage.prototype = {
 
     parseStorageDirSync: function() {
         var dir = this.options.dir;
-        var exists = fs.existsSync(dir);
+        var exists = fs.openSync(dir);
```

Comment:
* Wouldn't openSync throw an error here if the `dir` does not exists? you need the pass the flags and a `try/catch` here, where you set `catch (e) { exists = false }`